README.md

```diff
@@ -19,6 +32,10 @@ libratbag and initialize the build:
     cd libratbag
     meson builddir --prefix=/usr/
 
+This will inititialize libratbag so that it installs under `/usr`, i.e. it
+will overwrite the system installation. Choose a different prefix (e.g.
+`/usr/local`) if you want to install in parallel to the system version.
+
```

Comment:
I still think this should be reversed. Remove `--prefix=/usr/` and change the comment to tell the user to add it if he wants to overwrite the system's package. IMO it makes no sense using `--prefix=/usr/` as default, it breaks the system's package. `/usr/local/bin` should be first than `/usr/bin` in `$PATH` so if the user builds the project with the default prefix, that installation should be used by default, even if the user has installed libratbag via the package manager (installed to `/usr/bin`).

Reply:
Done and I punted a bit to [the wiki](https://github.com/libratbag/libratbag/wiki/Installation) where it's easier to update

Reply:
You could add something like this:

If you want to install to `/usr` instead of `/usr/local`, invoke meson with the following argument `--prefix=/usr`.